tests/test_streamable.py

```diff
@@ -41,97 +44,118 @@ def test_json_spend() -> None:
         "agg_sig_me": [["0x" + sig.hex(), "0x6d7367"]],
     }
 
+
 def test_from_json_spend() -> None:
 
     a = Spend(coin, ph, None, 0, [(ph2, 1000000, None)], [(sig, b"msg")])
 
-    b = Spend.from_json_dict({
-        "coin_id": "0x" + coin.hex(),
-        "puzzle_hash": "0x" + ph.hex(),
-        "height_relative": None,
-        "seconds_relative": 0,
-        "create_coin": [["0x" + ph2.hex(), 1000000, None]],
-        "agg_sig_me": [["0x" + sig.hex(), "0x6d7367"]],
-    })
+    b = Spend.from_json_dict(
+        {
+            "coin_id": "0x" + coin.hex(),
+            "puzzle_hash": "0x" + ph.hex(),
+            "height_relative": None,
+            "seconds_relative": 0,
+            "create_coin": [["0x" + ph2.hex(), 1000000, None]],
+            "agg_sig_me": [["0x" + sig.hex(), "0x6d7367"]],
+        }
+    )
     assert a == b
 
+
 def test_from_json_spend_set_optional() -> None:
 
     a = Spend(coin, ph, 1337, 0, [(ph2, 1000000, None)], [(sig, b"msg")])
 
-    b = Spend.from_json_dict({
-        "coin_id": "0x" + coin.hex(),
-        "puzzle_hash": "0x" + ph.hex(),
-        "height_relative": 1337,
-        "seconds_relative": 0,
-        "create_coin": [["0x" + ph2.hex(), 1000000, None]],
-        "agg_sig_me": [["0x" + sig.hex(), "0x6d7367"]],
-    })
+    b = Spend.from_json_dict(
```

Comment:
Are these just `black` fixes? You could probably get away with reverting them.

Reply:
yeah, I think so. reverted